README.md

```diff
@@ -583,18 +583,13 @@ like the `queries` folder of this plugin, e.g. `queries/{language}/{locals,highl
 Other modules may require additional queries such as `folding.scm`. You can find a
 list of all supported capture names in [CONTRIBUTING.md](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#parser-configurations).
 
-All queries found in the runtime directories will be combined.
-By convention, if you want to write a query, use the `queries/` directory,
-but if you want to extend a query use the `after/queries/` directory.
+Queries can be placed anywhere in your runtimepath under queries/<language>, with earlier directories taking precedence unless the queries are marked with ; extends; see :h treesitter-query.
```

Comment:
```suggestion
Queries can be placed anywhere in your runtimepath under queries/<language>, with earlier directories taking precedence unless the queries are marked with ; extends - see :h treesitter-query.
```

I'd use a hyphen or period - semicolon there can be confusing there as well and might seem like it's needed after extends

Reply:
Also nitpicked, wrap the `; extend`

Reply:
I will fix as suggested. FYI this sentence was copied literally from the main branch by @clason, I didn't change any single letter taking conflicts to consideration but these are all good to address. Anyway we cannot avoid any conflicts while rebasing (sorry about it) let's try to make the best form possible. I will modify text tomorrow.